MinimalMVVM/ViewModel/DelegateCommand.cs

```diff
@@ -12,18 +12,10 @@ public DelegateCommand(Action action)
             _action = action;
         }
 
-        public void Execute(object parameter)
-        {
-            _action();
-        }
+        public void Execute(object parameter) => _action();
 
-        public bool CanExecute(object parameter)
-        {
-            return true;
-        }
+        public bool CanExecute(object parameter) => true;
 
-#pragma warning disable 67
```

Comment:
This was added to stop ReSharper complaining about the unused event handler. It would appear that the latest version of ReShaper correctly no longer flags this.